src/DevChatter.Bot.Core/Commands/SimpleCommand.cs

```diff
@@ -24,6 +24,7 @@ public SimpleCommand(string commandText, string staticResponse, UserRole roleReq
 
         public string StaticResponse { get; protected set; }
         public UserRole RoleRequired { get; protected set; }
+        public string CommandType { get; protected set; } = "Default";
```

Comment:
I'm going to recommend against going this way, but to avoid having this stored in the database, you can make this a read-only property by removing the `protected set;` part.